riptide-spring-boot-autoconfigure/src/main/java/org/zalando/riptide/autoconfigure/FailsafePluginFactory.java

```diff
@@ -122,11 +121,12 @@ public static Plugin createRetryFailsafePlugin(
                     .withPolicy(new RetryRequestPolicy(policy.copy()
                             .handleIf(transientConnectionFaults()))
                             .withPredicate(alwaysTrue()))
+                    .withPolicy(new RetryRequestPolicy(policy.handle(RetryException.class)))
```

Comment:
Is there a risk here without doing a copy?

Reply:
Do you have something specific in mind? Every other use of `policy` is `copy()`'d, so I think this is fine.

Reply:
Unless the lack of copying is overlooked the next time there is a change needed here.